src/Client.php

```diff
@@ -58,6 +65,7 @@ public function __construct(string $endpoint, array $options = [], HttpClient $h
                 'unfurl_media' => true,
                 'allow_markdown' => true,
                 'markdown_in_attachments' => [],
+                'oauth_token' => null
```

Comment:
`token` should be enough.

Reply:
I prefer `oauth_token` as it matches what it's called in the Slack documentation and interface, and there are many other types of tokens available in Slack:

* OAuth Access Token
* Bot User OAuth Access Token
* Verification Token